Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1340,7 +1340,7 @@ def identityFS(wf):
     '-s':'HARVESTING:@standardValidation+@standardDQM+@miniAODValidation+@miniAODDQM', # todo: remove UP from label
     '--conditions':'auto:run2_mc', 
     '--mc':'',
-    '--era' : 'Run2_25ns',
+    '--era' : 'Run2_2016s',
```

Comment:
whoops - yes - I fixed that once… thanks

> On Apr 4, 2016, at 7:40 PM, Martin Grunewald <notifications@github.com> wrote:
> 
> In Configuration/PyReleaseValidation/python/relval_steps.py:
> 
> > @@ -1340,7 +1340,7 @@ def identityFS(wf):
> >      '-s':'HARVESTING:@standardValidation+@standardDQM+@miniAODValidation+@miniAODDQM', # todo: remove UP from label
> >      '--conditions':'auto:run2_mc', 
> >      '--mc':'',
> > -    '--era' : 'Run2_25ns',
> > +    '--era' : 'Run2_2016s',
> 
> Should this rather be Run2_2016 (instead of Run2_2016s)?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>